src/main/java/org/benetech/servicenet/matching/counter/UrlSimilarityCounter.java

```diff
@@ -14,48 +13,38 @@
     public static final String HTTPS = "HTTPS://";
     public static final String HTTP = "HTTP://";
     public static final String TRAILING = "/";
-    
-    @Value("${similarity-ratio.weight.url.equal-upper-cased}")
-    private float uppercasedWeight;
 
     @Override
     public float countSimilarityRatio(String url1, String url2, MatchingContext context) {
         if (StringUtils.isBlank(url1) || StringUtils.isBlank(url2)) {
             return NO_MATCH_RATIO;
         }
-        if (areNormalizedAndUpperCasedDifferent(url1, url2)) {
```

Comment:
this is used to calculate the match, you can't make this changes, because it will affect the matching algorithm.
revert all of the changes in this file